docs/git/private-dependencies.md

```diff
@@ -0,0 +1,29 @@
+When developing your app, you sometimes need to use a dependency from a private git repository.
+
+To do this, you need to upload a private git key as an environment variable, and make some changes to your Dockerfile.
+
+1. First make a Base64 encoded string of your private key.
+
+```bash
+cat /path/to/file | base64
+```
+
+Setup the value of this under the Environment variable key `SSH_PRIVATE_KEY` in Vapor Cloud 2
+
+2. Add the following lines to your `web.Dockerfile`
+
+```docker
+ARG ssh_prv_key=""
+
+# Authorize SSH Host
+RUN mkdir -p /root/.ssh && \
+    chmod 0700 /root/.ssh && \
+    ssh-keyscan github.com > /root/.ssh/known_hosts && \
```

Comment:
There is probably an artefact from dividing those commands into two `RUN` parts:
```suggestion
    ssh-keyscan github.com > /root/.ssh/known_hosts
```